### PR TITLE
Bump to GraphQL.NET 7.0.0

### PR DESCRIPTION
**Pull request by Shane32**

### CODE REVIEW

File: tests/Samples.Complex.Tests/BaseTest.cs [hunk 1]
```diff
@@ -15,6 +16,14 @@ protected BaseTest()
     {
         Host = Program.CreateHostBuilder(Array.Empty<string>())
              .ConfigureWebHost(webBuilder => webBuilder.UseTestServer())
+             .ConfigureServices(services =>
+             {
+                 services.AddGraphQL(b => b.ConfigureExecution((options, next) =>
```
Comment (sungam3r) [inline tests/Samples.Complex.Tests/BaseTest.cs:21]: How did it work before without `AddGraphQL`?

Comment (Shane32) [reply tests/Samples.Complex.Tests/BaseTest.cs:21]: It runs `AddGraphQL` within the `Program.cs` file.  For some reason the change to `UseApolloTracing` now started returning tracing results in the tests (which it should, I believe).  So this call to `AddGraphQL` serves only to disable metrics for the purposes of the tests, leaving the sample project in tact.

Comment (Shane32) [reply tests/Samples.Complex.Tests/BaseTest.cs:21]: Oh, the reason is obvious -- the call to ConfigureExecutionOptions sets EnableMetrics, which now occurs prior to `UseApolloTracing`.  I should have changed `UseApolloTracing()` to `UseApolloTracing(false)`, as the sample is now incorrect.

Comment (Shane32) [reply tests/Samples.Complex.Tests/BaseTest.cs:21]: #903

**Pull request merged**